src/main/resources/TotemCode/TotemBlockClass.xml

```diff
@@ -302,7 +306,11 @@
     <source>
       <cache>0</cache>
       <classname/>
-      <customDisplay>{{include document="xwiki:XWiki.SuggestDisplay" /}}</customDisplay>
+      <customDisplay>{{velocity}}
+#set($SuggestDisplayItemsEditable = true)
+#set($SuggestDisplayEditModeSpace = "Totem")
+{{/velocity}}
+{{include document="xwiki:XWiki.SuggestDisplay" /}}</customDisplay>
```

Comment:
Why do you force the document to be on the main wiki? You prevent MSD to be installed on the current wiki.

The `document` parameter is deprecated (http://extensions.xwiki.org/xwiki/bin/view/Extension/Include+Macro).

Please do:
```
{{include reference="XWiki.SuggestDisplay" /}}
```